docs/src/pages/components/masonry/ImageMasonry.js

```diff
@@ -0,0 +1,96 @@
+/* eslint-disable @typescript-eslint/no-use-before-define */
+import * as React from 'react';
+import Masonry from '@material-ui/lab/Masonry';
+import MasonryItem from '@material-ui/lab/MasonryItem';
+
+export default function ImageMasonry() {
+  return (
+    <Masonry
+      columns={{ xs: 3, sm: 5 }}
```

Comment:
Having played with it locally, I wonder if `{ xs: 3, md: 4 }` might be the best combination?